riscv/src/continuations/bootloader.rs

```diff
@@ -34,10 +35,13 @@ pub fn shutdown_routine_upper_bound(num_pages: usize) -> usize {
     let constant_overhead = 6 + REGISTER_NAMES.len() - 1;
 
     // For each page, we have to:
+    // TODO is 14 still the true number?
     // - Start the page loop (14 instructions)
     // - Load all words of the page
     // - Invoke the hash function once every 4 words
+    // TODO is 4 still the true number?
     // - Assert the page hash is as claimed (4 instructions)
```

Comment:
Guess now it's 8

Reply:
not sure. There are 2 4s here:

    // - Invoke the hash function once every 4 words
this one I think stays 4?

    // - Assert the page hash is as claimed (4 instructions)
I guess 8 words, but not sure how many instructions. Will change this one to 8.